### ISSUE TITLE
emitted PNGs are … weird

### ISSUE THREAD

**Issue by glyph** [sjml/SimulatorGenerator#3]
if I look at the output of SimulatorGenerator, the image looks fine, but,
1. `pngcrush` crashes if I run it, and produces an empty output file
2. `convert` produces a bunch of warnings, like `convert: iCCP: profile 'icc': 'desc': ICC profile tag start not a multiple of 4 `box.png' @ warning/png.c/MagickPNGWarningHandler/1656.`

I am just wondering what's up with that.

**Comment by sjml** [comment]
Woah! 

(a) I had no idea people were actually using SimGenerator. Cool! 
(b) Sorry the images are wonky. The code here doesn't do anything low-level with the image format, so I'm liable to blame the underlying libraries. But I'd be surprised if ImageMagick produced bad output; it's a pretty thoroughly used and tested piece of software. 

Are you able to share the images that are causing problems? And maybe what inputs are leading to them? 

How comfortable are you with editing the code to see if you emit, like, a JPEG or TIFF they end up better?

**Comment by glyph** [comment]
https://twitter.com/glyph/status/792543009350967296

**Comment by glyph** [comment]
I'll mess around with the output parameters; I mainly filed this to see if it was a known problem.

**Comment by sjml** [comment]
Yeah, I never did anything with the emitted files other than immediately post them to Twitter, so never encountered any kind of post-processing issues.
